### PR TITLE
Move the live webinars to featured

### PR DESCRIPTION
**Pull request by tatevikteroyan**
Move the live webinars to featured

### CODE REVIEW

**Comment by PRMerger12** [comment]
@tatevikteroyan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by tatevikteroyan** [comment]
sign-off

File: powerbi-docs/fundamentals/webinars.md [hunk 1]
```diff
@@ -16,20 +16,20 @@ Register for our upcoming live webinars or watch our recorded sessions on-demand
...
+
+## Featured webinars
+
+Get started with these popular on-demand webinars:
+
 **Webinar Series: Mastering Data Modeling with Power BI  
 Episode 1 - Data Modeling 101: Increasing the Impact of Power BI**  
 by Jeroen ter Heerdt and Marc Lelijveld  
-[Register to watch on March 17 10AM-11AM PST](https://info.microsoft.com/ww-Landing-Mastering-Data-Modeling-with-Power-BI.html?LCID=EN-US)
+[Watch now](https://info.microsoft.com/ww-Landing-Mastering-Data-Modeling-with-Power-BI.html?LCID=EN-US)
```
Comment (maggiesMSFT) [inline powerbi-docs/fundamentals/webinars.md:28]:
```suggestion
[Watch now](https://info.microsoft.com/ww-Landing-Mastering-Data-Modeling-with-Power-BI.html)
```

File: powerbi-docs/fundamentals/webinars.md [hunk 2]
```diff
@@ -16,20 +16,20 @@ Register for our upcoming live webinars or watch our recorded sessions on-demand
...
 **Quickstart Guide to Navigating Power BI**  
 by Miguel Martinez  
-[Register to watch on March 24 10AM-11AM PST]( https://info.microsoft.com/ww-Landing-Quickstart-Guide-to-Navigating-Power-BI.html?LCID=EN-US)  
-
-[Upcoming webinars from the Power BI community](https://powerbi.microsoft.com/blog/tag/community-webinars/?Is=Website)
-
-## Featured webinars
-
-Get started with these popular on-demand webinars:
+[Watch now]( https://info.microsoft.com/ww-Landing-Quickstart-Guide-to-Navigating-Power-BI.html?LCID=EN-US)
```
Comment (maggiesMSFT) [inline powerbi-docs/fundamentals/webinars.md:32]:
```suggestion
[Watch now](https://info.microsoft.com/ww-Landing-Quickstart-Guide-to-Navigating-Power-BI.html)
```

**Review by maggiesMSFT — CHANGES_REQUESTED** [review]

**Comment by PRMerger8** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by maggiesMSFT — APPROVED** [review]

**Comment by maggiesMSFT** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@maggiesMSFT: I'm sorry - only the author of this article, @kfollis, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**